plugins/ssh-plugin/src/node/ws-master-http-client.ts

```diff
@@ -0,0 +1,85 @@
+/*********************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { AxiosInstance, AxiosPromise, AxiosRequestConfig, default as axios } from 'axios'
+
+/**
+ * Plain wrapper around {@link AxiosInstance} class that is configured to
+ * have workspace API endpoint location as base URL. The value of API endpoint
+ * location is fetched out of environment variable called 'CHE_API'. If there
+ * is no such variable set the client tries to continue with default property
+ * values, however no correct request addressing is guaranteed in this case.
+ *
+ */
+export class WsMasterHttpClient {
+
+    private readonly ISSUE = 'Can\'t properly configure workspace master http client';
+    private readonly REASON = 'CHE_API environment property is undefined';
+
+    private readonly axiosInstance: AxiosInstance;
+
+    constructor(cheApi: string | undefined) {
```

Comment:
shouldn't be part of `@eclipse-che/plugin` ? so needs to use Axios there.
IMHO SSH should be provided by the API of che.

like https://github.com/eclipse/che-theia/blob/master/extensions/eclipse-che-theia-plugin/src/che-proposed.d.ts#L24 (for workspace namespace, should be a ssh namespace as this is part of WsMaster)